kalite/distributed/static/js/distributed/topics/views.js

```diff
@@ -307,12 +307,16 @@ window.TopicContainerInnerView = BaseView.extend({
         return this;
     },
 
-    add_new_entry: function(entry) {
+    add_new_entry: function(entry, no_append) {
         var view = new SidebarEntryView({model: entry});
         this.listenTo(view, "hideSidebar", this.hide_sidebar);
         this.listenTo(view, "showSidebar", this.show_sidebar);
         this._entry_views.push(view);
-        this.$(".sidebar").append(view.render().$el);
+        if (!no_append) {
+            this.$(".sidebar").append(view.render().$el);
+        } else {
+            view.render();
```

Comment:
No, it won't, because the view.$el has not been inserted into the DOM yet (unless no_append is false).